modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/capability/xml/schema/builder/SchemaBuilder.java

```diff
@@ -940,6 +969,11 @@ Annotation createDocAnnotation(String content)
         return annotation;
     }
 
+    MetadataType load(Class<?> clazz)
```

Comment:
Will this have other uses in the future? I only see 2 usages, both for String.class